### PR TITLE
created Question model

### LINKED ISSUE #19: Create "Question" model under "questions" app
Refer [here](https://drive.google.com/file/d/1rjZe-7mgDX1MGz6FY9YgBMY7_IpSH26-/view?usp=share_link) document for "Question" model creation in questions/models.py

Avoid using prefix of table name in the table fields. For eg. In the PDF if "question_name" is written then use it as "name" only.

Refer to DSA_Q table in pdf for the model.

Also generate the migrations for this app.

### PR DESCRIPTION
**Pull request by animesh288**
### Description
Created Question model under questions app and generated migrations for the app.

**Type of Change:** 
- Code

**Code/Quality Assurance Only** 

- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [ ] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: questions/models.py [hunk 1]
```diff
@@ -1,3 +1,18 @@
 from django.db import models
 
-# Create your models here.
+class Question(models.Model):
+    id=models.IntegerField(primary_key=True)
```
Comment (sahil9001) [inline questions/models.py]: Not required, `Model` already has an `id` field.

File: questions/models.py [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+    topic=models.CharField(max_length=200)
```
Comment (sahil9001) [inline questions/models.py]: `ForeignKey` field with `Topic` model.
Dependency : Topic Model

File: questions/models.py [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+    name=models.CharField(max_length=200)
+    codechefLink=models.CharField(max_length=200)
+    codeforcesLink=models.CharField(max_length=200)
+    gfgLink=models.CharField(max_length=200)
+    leetcodeLink=models.CharField(max_length=200)
+    hackerrankLink=models.CharField(max_length=200)
```
Comment (sahil9001) [inline questions/models.py]: Make this `URLField`

File: questions/models.py [hunk 4]
```diff
@@ -1,3 +1,18 @@
...
+    global_check=models.BooleanField(default=False)
```
Comment (sahil9001) [inline questions/models.py]: Can you explain what's this for? Seems irrelevant.

File: questions/models.py [hunk 5]
```diff
@@ -1,3 +1,18 @@
...
+    level=models.CharField(max_length=20)
```
Comment (sahil9001) [inline questions/models.py]: Make it `ChoiceField` with difficulty as 'easy' , 'medium' and 'hard'.

File: questions/models.py [hunk 6]
```diff
@@ -1,3 +1,18 @@
...
+    mentor_id=models.IntegerField()
```
Comment (sahil9001) [inline questions/models.py]: Will be a `ForeignKey` when `Mentor` model is completed.

**Review by sahil9001 — CHANGES_REQUESTED** [review]

File: questions/models.py (hunk 4 above)
Comment (animesh288) [reply questions/models.py]: I added it following [this](https://drive.google.com/file/d/1rjZe-7mgDX1MGz6FY9YgBMY7_IpSH26-/view) schema.
Updated in the latest commit.

File: questions/models.py (hunk 6 above)
Comment (animesh288) [reply questions/models.py]: Commented out this field. Will update once the Mentor model is completed

**Comment by prakhar-asaiya** [comment]
add a __str__ class as well @animesh288 .

**Comment by prakhar-asaiya** [comment]
add a '__str__' method as well @animesh288.

**Comment by prakhar-asaiya** [comment]
add a __'__str__'__ class as well @animesh288.

**Comment by ayushgoswamiabr** [comment]
Seems Ok to me @sahil9001 I think all the requested changes are done will you take a look and approve it

**Review by sahil9001 — APPROVED** [review]

**Review by ayushgoswamiabr — APPROVED** [review]

**Pull request merged**